src/components/pager/pager.twig

```diff
@@ -0,0 +1,103 @@
+{#
+/**
+ * @file
+ * Theme override to display a pager.
+ *
+ * Available variables:
+ * - items: List of pager items.
+ *   The list is keyed by the following elements:
+ *   - first: Item for the first page; not present on the first page of results.
+ *   - previous: Item for the previous page; not present on the first page
+ *     of results.
+ *   - next: Item for the next page; not present on the last page of results.
+ *   - last: Item for the last page; not present on the last page of results.
+ *   - pages: List of pages, keyed by page number.
+ *   Sub-sub elements:
+ *   items.first, items.previous, items.next, items.last, and each item inside
+ *   items.pages contain the following elements:
+ *   - href: URL with appropriate query parameters for the item.
+ *   - attributes: A keyed list of HTML attributes for the item.
+ *   - text: The visible text used for the item link, such as "‹ Previous"
+ *     or "Next ›".
+ * - current: The page number of the current page.
+ * - ellipses: If there are more pages than the quantity allows, then an
+ *   ellipsis before or after the listed pages may be present.
+ *   - previous: Present if the currently visible list of pages does not start
+ *     at the first page.
+ *   - next: Present if the visible list of pages ends before the last page.
+ *
+ * @see template_preprocess_pager()
+ */
+#}
+{% set pager__base_class = 'pager' %}
+
+{% set pager__attributes = pager__attributes|default({
+  'class': bem(pager__base_class, pager__modifiers, pager__blockname),
+  'aria-label': pagination_id|default('Pagination'|t),
+  'role': 'navigation'
+}) %}
+
+{% if items %}
+  <nav {{ add_attributes(pager__attributes) }}>
+    <ul {{ bem('items', [], pager__base_class, ['js-pager__items']) }}>
+      {# Print first item. #}
+      {% if items.first %}
+        <li {{ bem('item', ['first'], pager__base_class) }}>
+          <a {{ bem('link', ['first'], pager__base_class) }} href="{{ items.first.href }}" title="{{ 'Go to first page'|t }}" {{ items.first.attributes|without('href', 'title', 'rel') }}>
+            <span {{ bem('visually-hidden') }}>{{ 'First page'|t }}</span>
+            <span aria-hidden="true">{{ items.first.text|default('First'|t) }}</span>
+          </a>
+        </li>
+      {% endif %}
+      {# Print previous item if we are not on the first page. #}
+      {% if items.previous %}
+        <li {{ bem('item', ['prev'], pager__base_class) }}>
+          <a {{ bem('link', ['prev'], pager__base_class) }} href="{{ items.previous.href }}" title="{{ 'Go to previous page'|t }}" rel="prev"{{ items.previous.attributes|without('href', 'title', 'rel') }}>
+            <span {{ bem('visually-hidden') }}>{{ 'Previous page'|t }}</span>
```

Comment:
If we are only adding a single class, should the span be `<span class="visually-hidden">` without the bem function? No need to change it. I'm just curious about the answer to the question.

Reply:
@amazingrando Oh! I just copy-paste what it was already of the compound components. That's why it's like that, but you're right